internal/cli/publish.go

```diff
@@ -243,23 +249,18 @@ func PublishCmd(ctx context.Context, outputRefs string, archs []types.Architectu
 
 // publishImage publishes a specific architecture image
 func publishImage(bc *build.Context, layerTarGZ string, arch types.Architecture) (imgDigest name.Digest, img coci.SignedImage, err error) {
-	// We only tag the images if we're building a single one
-	imageTags := []string{}
-	if len(bc.ImageConfiguration.Archs) == 1 {
```

Comment:
I'm curious why we need this part of the change, or whether it regresses something 🤔

Reply:
Do you think it's better if I dont pass it to the tagging function and build the reference by hand? I did passed it because it was returning an empty string .

Reply:
I don't have enough context here, so I'd defer to @kaniini

Reply:
OK, I thought you had a specific problem in mind. I think we're fine, tests are ok and I've been using this branch today ok.